components/fxa-client/src/http_client.rs

```diff
@@ -309,7 +333,36 @@ impl FxAClient for Client {
             .header(header_names::AUTHORIZATION, bearer_token(refresh_token))?
             .header(header_names::CONTENT_TYPE, "application/json")?
             .body(body.to_string());
-        Self::make_request(request)?;
+        // I am a really bad way to handle this
+        // I exist only because Tarik is experimenting :)
+        // I will be replaced with a better approach
+        match Self::make_request(request) {
```

Comment:
I think this is on the right track, looking at the code required to just short circuit the command route, we should just upgrade our `make_request` to be ubiquitous for other FxA endpoints.